src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -12,13 +16,57 @@ public class ListCommand extends Command {
 
     public static final String COMMAND_WORD = "list";
 
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Lists relevant contacts from the address book. "
+            + "Parameters: "
+            + "[" + PREFIX_TAG + "TAG]..."
+            + "[" + PREFIX_WARD + "WARD]\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_TAG + "Fall-risk "
```

Comment:
Should we decide whether we want to stick to FallRisk or Fall-risk?

Reply:
I will change it to FallRisk. Thanks!